cmd/kv_write.go

```diff
@@ -96,11 +101,17 @@ func kvWrite(*cobra.Command, []string) {
 		logrus.WithError(err).Fatal("Failed to create flow contract")
 	}
 
+	var fee *big.Int
+	if kvWriteArgs.fee > 0 {
+		feeInA0GI := big.NewFloat(kvWriteArgs.fee)
+		fee, _ = feeInA0GI.Mul(feeInA0GI, big.NewFloat(1e18)).Int(nil)
```

Comment:
how about let users to convert this and we only accept neuron?

Reply:
not sure, it might require the user to input a number that is a dozen digits long